content/applications/general/calendars/google/google_calendar_credentials.rst

```diff
@@ -2,89 +2,130 @@
-    :align: center
+.. tip::
+   Give the API Project a clear name like "Odoo Sync" so you can easily find it.
 
-- Enable the API.
+Then, open the API Project and click :guilabel:`Enable APIs and Services`.
 
-.. image:: media/google_calendar_credentials03.png
-    :align: center
+.. image:: google_calendar_credentials/enable_apis_services.png
+   :align: center
+   :alt: Enable APIs and Services on the API Project.
 
-- Select or create an API project to store the credentials if not yet done 
-  before. Give it an explicit name (e.g. Odoo Sync).
+After that, search for *Google Calendar API* using the search bar and select **Google Calendar
+API** from the search results. Click :guilabel:`Enable`.
 
-- Create credentials.
+.. image:: google_calendar_credentials/enable_google_cal_api.png
+   :align: center
+   :alt: Enable the Google Calendar API.
 
-.. image:: media/google_calendar_credentials04.png
-    :align: center
+Create credentials
+------------------
 
-- Select *Web browser (Javascript)* 
-  as calling source and *User data* as kind of data.
+Now that you have created your API project and enabled the Google Calendar API, you need to create
+credentials. Begin by clicking :guilabel:`Create Credentials`. Google will then guide you through
+four steps to create your API credentials.
 
-.. image:: media/google_calendar_credentials05.png
-    :align: center
+- In the first step, **Credential Type**, select the *Google Calendar API* and *User Data* options.
+  Then, click :guilabel:`Next`.
 
-- Then you can create a Client ID.
-  Enter the name of the application (e.g. Odoo Calendar) and the allowed pages on 
-  which you will be redirected. The *Authorized JavaScript origin* is your 
-  Odoo's instance URL. The *Authorized redirect URI* is your Odoo's instance 
-  URL followed by '/google_account/authentication'.
+.. image:: google_calendar_credentials/credential_type.png
+   :align: center
+   :alt: Select Google Calendar API and User Data for the Credential Type.
 
-.. image:: media/google_calendar_credentials06.png
-    :align: center
+- In the **OAuth Consent Screen** step, type *Odoo* in the :guilabel:`App name` field, select your
+  email address for the :guilabel:`User support email` field, and type your email address for the
+  :guilabel:`Developer contact information` section. Click :guilabel:`Save and Continue`.
 
-- Go through the Consent Screen step by entering a product name 
-  (e.g. Odoo Calendar). Feel free to check the customizations options 
-  but this is not mandatory. The Consent Screen will only show up when you 
-  enter the Client ID in Odoo for the first time.
+- Skip the **Scopes** step by clicking :guilabel:`Save and Continue`.
 
-- Finally you are provided with your **Client ID**. Go to *Credentials* to 
-  get the **Client Secret** as well. Both of them are required in Odoo.
+- In the **OAuth Client ID** step, select *Website application* for the :guilabel:`Application
```

Comment:
Stopping here.
Please revise the entire document further to include `:guilabel:`for any/all on-screen selection items, headings, fields, options, buttons, etc.
```suggestion
- In the :guilabel:`OAuth Client ID` step, select :guilabel:`Website application` for the :guilabel:`Application
```